### PR TITLE
SI-8085 Fix BrowserTraverser for package objects

### PR DESCRIPTION
**Pull request by retronym**
Review by @dotta

### CODE REVIEW

File: test/files/presentation/t8085/Test.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import scala.tools.nsc.interactive.tests.InteractiveTest
+import scala.reflect.internal.util.SourceFile
+import scala.tools.nsc.interactive.Response
+
+object Test extends InteractiveTest {
+
+  override def execute(): Unit = {
+    // loadSourceAndWaitUntilTypechecked("package.scala") // <-- uncomment this line and the test succeed
```
Comment (dotta) [inline test/files/presentation/t8085/Test.scala]: Is it worth keeping this? The comment is not misleading...

File: test/files/presentation/t8085b/Test.scala [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import scala.tools.nsc.interactive.tests.InteractiveTest
+import scala.reflect.internal.util.SourceFile
+import scala.tools.nsc.interactive.Response
+
+object Test extends InteractiveTest {
+
+  override def execute(): Unit = {
+    // loadSourceAndWaitUntilTypechecked("package.scala") // <-- uncomment this line and the test succeed
```
Comment (dotta) [inline test/files/presentation/t8085b/Test.scala]: Same comment as in https://github.com/scala/scala/pull/3282/files#r8429538

**Comment by dotta** [comment]
LGTM

**Comment by dotta** [comment]
Thanks a lot Jason! Really great XMas present ;-)

File: test/files/presentation/t8085/Test.scala (hunk 1 above)
Comment (retronym) [inline test/files/presentation/t8085/Test.scala]: Removed.

**Pull request merged**